torch/testing/_internal/common_methods_invocations.py

```diff
@@ -108,7 +123,9 @@ def __init__(self,
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
                  sample_inputs_func=None,  # function to generate sample inputs
                  aten_name=None,  # name of the corresponding aten:: operator
-                 supports_sparse=False  # supported for sparse
+                 aliases=None,  # list of aliases, e.g. torch.absolute -> torch.abs
+                 supports_sparse=False,  # whether the op correctly handles sparse input
+                 sparse_op_info=None,  # sparse op info as dict, e.g. flag that op supports inplace on uncoalesced
```

Comment:
Instead of "sparse_op_info" I think we can just inline the sparse properties like any OpInfo property. So a property like `supports_inplace_on_uncoalesced` would just be any other property.

If the number of sparse-specific properties gets too large then we can revisit having a separate SparseOpInfo class with a pointer to the original OpInfo inside it.